File: power calculator.cpp
#include <iostream>

int main() {
    int base, exp;
    std::cout << "Enter base: ";
    std::cin >> base;
    std::cout << "Enter exponent: ";
    std::cin >> exp;

    long long result = 1;
    for (int i = 0; i < exp; ++i) {
        result *= base;
    }

    std::cout << "Result: " << result << std::endl;
    return 0;
}

File: loops foundation/power calculator.py
baseNum =  int(input("Enter the base number: "))
exponent = int(input("Enter the exponent: "))
result = 1
i = 1
while i <= exponent:
    result = result*baseNum
    i = i+1
print(f"{baseNum} raised to the power of {exponent} is {result}")
